cmake/pika_setup_hip.cmake

```diff
@@ -12,6 +12,22 @@ if(PIKA_WITH_HIP AND NOT TARGET roc::rocblas)
     )
   endif(PIKA_WITH_CUDA)
 
+  # Check HIP standard
+  if(DEFINED CMAKE_HIP_STANDARD AND NOT CMAKE_HIP_STANDARD STREQUAL
+                                     PIKA_WITH_CXX_STANDARD
+  )
+    pika_error(
+      "You've set CMAKE_HIP_STANDARD to ${CMAKE_HIP_STANDARD} and PIKA_WITH_CXX_STANDARD to ${PIKA_WITH_CXX_STANDARD}. Please unset CMAKE_HIP_STANDARD."
+    )
+  endif()
```

Comment:
This should be updated to reflect the changes in #396.

Reply:
yes already in todo list thanks!